frontend/star-tides/src/test.ts

```diff
@@ -0,0 +1,25 @@
+// This file is required by karma.conf.js and loads recursively all the .spec and framework files
```

Comment:
I don't know how to run them. For now let's leave it out and we can add it in later how you see fit.

Reply:
Try this:

```bash
$ cd frontend/star-tides && npm i && ng test --watch=false --browsers=ChromeHeadless
```

Reply:
After setting it up, we should exclude it from the docker container. Angular tests rely on chromeheadless and I don't want to install chrome into the docker container. We can however have this as a pre-commit hook. It may make sense to separate the frontend code into a different repo.